spec/helpers/find/provider_helper_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Find::ProviderHelper, type: :helper do
+  describe "#select_provider_options" do
+    let(:providers) { build_list(:provider, 3) }
+
+    subject { select_provider_options(providers) }
+
+    it "returns select provider options" do
+      expect(subject.map(&:id)).to eql([""] + providers.map { |p| "#{p.provider_name} (#{p.provider_code})" })
```

Comment:
Could we put the expected values behind a meaningful method name so it reads a little better?

Reply:
Fixed